src/View/Helper/PaginatorHelper.php

```diff
@@ -820,9 +822,10 @@ protected function _firstNumber($ellipsis, $params, $start, $options)
     protected function _lastNumber($ellipsis, $params, $end, $options)
     {
         $out = '';
+        $last = is_int($options['last']) ? $options['last'] : 0;
```

Comment:
I think so.
I just tested on an old Cake2 app (V 2.4.0), and it seems that there's is also a bug related to that, but a bit different.

#### With an integer :
```php
<?= $this->Paginator->numbers(array(
    'modulus' => 2,
    'tag' => 'li',
    'currentTag' => 'a',
    'separator' => '',
    'ellipsis' => '<li><a>...</a></li>',
    'first' => 1, //With an integer
    'last' => 1 // With an integer
));?>
```
Result : It look good, the `ellipsis` is added correctly.
![9a5201daae91dd515f4201147ef207c7](https://cloud.githubusercontent.com/assets/8210023/11782576/028ce4e0-a272-11e5-8f8f-09eeb33aa18d.png)

#### With a string number :
```php
<?= $this->Paginator->numbers(array(
    'modulus' => 2,
    'tag' => 'li',
    'currentTag' => 'a',
    'separator' => '',
    'ellipsis' => '<li><a>...</a></li>',
    'first' => '1', // With a string number
    'last' => '11' // With a string number
));?>
```
Result : The `ellipsis` are not rendered, **and** it display all the page after the current page
![bbd623529159615792c724d4005082a8](https://cloud.githubusercontent.com/assets/8210023/11782645/70de1932-a272-11e5-8ccd-130728875683.png)


#### With a *real* string :
```php
<?= $this->Paginator->numbers(array(
    'modulus' => 2,
    'tag' => 'li',
    'currentTag' => 'a',
    'separator' => '',
    'ellipsis' => '<li><a>...</a></li>',
    'first' => 'abc', // With a 'real' string
    'last' => 'abc' // With a 'real' string
));?>
```

Result : The `ellipsis` are not rendered, **but** it doesn't display all the page after the current page.
![d3a433ac79a73918d9d31427da9b3179](https://cloud.githubusercontent.com/assets/8210023/11782703/e138f6de-a272-11e5-97a6-19c964af87be.png)

So for me yes, there's a bug. The `1` & `'1'` seems to be identically the same by Cake2, probably by this condition : https://github.com/cakephp/cakephp/blob/2.7/lib/Cake/View/Helper/PaginatorHelper.php#L801

Also, the `ellipsis` are not rendered with a string. I don't know if it's intended or not. But if it's not intended, then it's another bug.